tomcat-base/src/shared/main/java/eu/xenit/alfresco/tomcat/embedded/tomcat/TomcatFactory.java

```diff
@@ -190,9 +190,9 @@ private void redirectLog4j(Path path) {
             try (var reader = Files.newBufferedReader(log4JPropertiesPath)) {
                 properties.load(reader);
                 properties.setProperty("log4j.rootLogger", "error, Console, jmxlogger1");
-                properties.setProperty("log4j.appender.Console.layout", "biz.paluch.logging.gelf.log4j.GelfLayout");
+                properties.setProperty("log4j.appender.Console.layout", "eu.xenit.json.log4j.JsonFormatter");
                 properties.setProperty("log4j.appender.Console.layout.AdditionalFields",
```

Comment:
same here. This is a Xenit specific JSON formatter. Type and application are mandatory. I expect something like this:
properties.setProperty("log4j.appender.Console.layout.Type","application");
properties.setProperty("log4j.appender.Console.layout.Component", path.getFileName());